docs/settings/alert-action-settings.asciidoc

```diff
@@ -199,6 +199,22 @@ Specifies the time allowed for requests to external resources. Requests that tak
 +
 For example, `20m`, `24h`, `7d`, `1w`. Default: `60s`.
 
+`xpack.actions.run.maxAttempts` {ess-icon}::
```

Comment:
This says that `maxAttempts` indicates the number of retries, but when verifying it seems that it means the number of tries. When I set `maxAttempts` to 1, I only see the action run once and then not retried, when I would expect it to see it retried 1 time.

Reply:
I set `maxAttempts: 1` and force an email error. After the first action run, I see this log:

```
[2022-12-21T13:22:25.464-05:00][ERROR][plugins.actions.email] Error: email action error
    at Object.executor (/Users/ying/Code/kibana/x-pack/plugins/stack_connectors/server/connector_types/stack/email/index.ts:230:9)
    at Object.wrapper [as executor] (/Users/ying/Code/kibana/node_modules/lodash/lodash.js:5255:19)
    at /Users/ying/Code/kibana/x-pack/plugins/actions/server/lib/action_executor.ts:154:38
[2022-12-21T13:22:25.464-05:00][WARN ][plugins.actions.email] action execution failure: .email:gmail: email: my gmail: an error occurred while running the action: email action error; retry: true
```

which says `retry: true`. Since maxAttempts is meant to be max retries, I would expect this to be retried once and the logs indicate it will be retried. However, the action task is deleted after this and the action is never retried.

Reply:
Good catch, @ymao1! We should fix our docs to mention "attempt" instead of "retries". Basically this value confirms how many times an action will run (attempt) before aborting it.

Reply:
Agreed that the `getRetry` / `retry: true` stuff can be confusing 🙈 +1 to a follow up if we didn't break it.

Reply:
Yea, I don't think this is caused in this PR. A followup issue is fine

Reply:
Replaced retry with attempt :)
And checked the `retry: true` in the response, action executor returns it when there is an error other than a validation error. I think it means retryable not "the last execution was a retry".